src/spatio_temporal_voxel_layer.cpp

```diff
@@ -545,12 +545,11 @@ void SpatioTemporalVoxelLayer::activate(void)
   RCLCPP_INFO(logger_, "%s was activated.", getName().c_str());
 
   observation_subscribers_iter sub_it = _observation_subscribers.begin();
-  for (; sub_it != _observation_subscribers.end(); ++sub_it) {
-    (*sub_it)->subscribe();
```

Comment:
Why did you remove the subscribe function? We need this to start processing things for any topics at all regardless of enabled status.

Reply:
When the subscriber is created, it automatically subscribes to the topic because we pass all the needed parameters, see
https://github.com/SteveMacenski/spatio_temporal_voxel_layer/blob/94e8439f7d1ff7038ce9c1138df5d2e7c743dbc2/src/spatio_temporal_voxel_layer.cpp#L315
and
https://github.com/ros2/message_filters/blob/de46fa8086f4b5586ea3ea80fcbbce8984ad15d2/include/message_filters/subscriber.h#L182